submissions/IRETIOGEDENGBE/HTML/Homepage.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="homepage.css" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Homepage</title>
+</head>
+<body>
+    <header>
+        <h1>
+                Aloha Store   
+        </h1>
+        <div class="dropdown-ctn">
+            <a href="signup.html">Sign up</a>
+            <a href="#">Login</a>
+            <a href="#">Track Orders</a>
+            <a href="#">Wish List</a>
+            </div>
+    </header>
+            <main> 
+                <div>
+                        <img src="C:\Users\Oluwasanmi/Desktop\work\Dufuna\Dufuna-html\hungcloth.jpg" alt="promo!!!" class="img1">
```

Comment:
@Ireti203 You need to use a relative path for your images, this would only work on your own computer. Checkout @olumid300's [PR](#76) to see how to do this.